### ISSUE TITLE
Slice failed when using support blocker

### ISSUE THREAD

**Issue by Yankeeki2024** [Ultimaker/Cura#18808]
### Cura Version

5.6 or 5.7

### Operating System

Windows 11

### Printer

Ulitimaker

### Describe model location

any project

### Describe your model

dino

### Add your .zip here ⬇️

why cant I attach a project file?  doesn't accept .3mf?

**Comment by Yankeeki2024** [comment]
[UMS5_Main_Dino.zip](https://github.com/Ultimaker/Cura/files/14857144/UMS5_Main_Dino.zip)

**Comment by HellAholic** [comment]
[UMS5_Main_Dino.zip](https://github.com/Ultimaker/Cura/files/14857434/UMS5_Main_Dino.zip)
Can you check and see if you are able to slice this one?

**Comment by Yankeeki2024** [comment]
yes.  But as soon as I try to add a support block I get an error.

![image](https://github.com/Ultimaker/Cura/assets/165942871/ec804394-d450-46b5-880f-5b973691ee9a)

**Comment by Yankeeki2024** [comment]
I also followed instructions above and deleted and all instances and reinstalled version 5.3.  Issue did not resolve.

**Comment by HellAholic** [comment]
Support blocker is a mesh and by default gets assigned to the first extruder.
You are disabling the first extruder on your printer and adding support blockers.
Although a support blocker is not printed it is a mesh, the initial check for whether any type of mesh is assigned to a disabled extruder fails.
You can either switch the extruder of the Support blocker meshes to extruder 2, or enable extruder 1 and have not touch support blockers.
In either case it will slice.

**Comment by Yankeeki2024** [comment]
You are absolutely correct.  I've been using nozzle 2 to use up some old PLA.

Thanks

**Comment by ro-mak** [comment]
> Support blocker is a mesh and by default gets assigned to the first extruder. You are disabling the first extruder on your printer and adding support blockers. Although a support blocker is not printed, it is a mesh; the initial check for whether any type of mesh is assigned to a disabled extruder fails. You can either switch the extruder of the Support blocker meshes to extruder 2, or enable extruder 1 and not touch support blockers. In either case it will slice.

thanks, changing the extruder for the eraser helped
I think it will be great to either automatically set the available extruder or at least write a warning message describing the reason why Cura is not able to slice
